pkg/runtime/context/context.go

```diff
@@ -30,6 +30,8 @@ import (
 const (
 	_flagMaster cFlag = 1 << iota
 	_flagSlave
+	_flagRead
```

Comment:
`_flagMaster ` and `_flagSlave ` are Useless?  If so, suggest remove them.

Reply:
yes,  `_flagMaster` and `_flagSlave` is useless，I will remove them latter